src/bmp/decoder.rs

```diff
@@ -1344,8 +1340,20 @@ impl<'a, R: 'a + Read + Seek> ImageDecoderExt<'a> for BMPDecoder<R> {
 
         let data = data?;
 
-        image::load_rect(x, y, width, height, buf, progress_callback, self, |_, _| unreachable!(),
-                         |_, buf| { buf.copy_from_slice(&data); Ok(buf.len()) })?;
+        image::load_rect(
+            x,
+            y,
+            width,
+            height,
```

Comment:
Can we teach it to keep `x, y, width, height` on one or two lines? This just takes a ton of space for virtually nothing.

Reply:
We can add `#[rustfmt::skip]` here.